packages/fxa-settings/src/components/BrandMessaging/en.ftl

```diff
@@ -0,0 +1,26 @@
+## Brand Messaging component
+## Used to show in product messaging about upcoming brand changes
+
+# This aria-label applies to the dismiss/close button of the banner
+# This text is for screen-readers
+brand-banner-dismiss-button =
+  .aria-label = Close
+
+# This message is displayed as the title element in the banner, prior to actually launching the new brand
+brand-prelaunch-title = Firefox accounts will be renamed Mozilla accounts on Nov 1
```

Comment:
```suggestion
brand-prelaunch-title = { -product-firefox-accounts } will be renamed { -product-mozilla-accounts } on Nov 1
```

We should use the brand term here for `Firefox accounts`. We should also create terms in `branding.ftl` for `Mozilla account` and `Mozilla accounts`. In addition, we'll want to include comments to make sure it matches the [content guidelines](https://www.figma.com/file/7nsOBSt3VTeCimDqavQT9J/Mozilla-Accounts-Content-Guidelines?type=design&node-id=1901-5540&mode=design&t=jRsPpgIIN6tWgvWV-0) around usage of Mozilla accounts.

E.g.

```
# "account" can and should be localized, "Mozilla" must be treated as a brand. Singular "Mozilla account" should be used in most cases.
-product-mozilla-account = Mozilla account
# "accounts" can and should be localized, "Mozilla" must be treated as a brand. Plural "Mozilla accounts" is used when referring to something affecting all Mozilla accounts, not just the individual's account.
-product-mozilla-accounts = Mozilla accounts
```

Reply:
Thanks for the call out. I've updated strings to use the brand terms.

Reply:
branding terms used.